### PR TITLE
feat: add staking flow tracking properties for first steps

### PR DESCRIPTION
**Pull request by mitchellv-ledger**
### 📝 Description

Adds "Step Starter" to tracking on first steps of staking flows on LLM.

### ❓ Context

- JIRA: [LIVE-9901] 

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [x] `npx changeset` was attached.
- [x] **Covered by automatic tests.** 
- [x] **Impact of the changes:** 
  - 

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-9901]: https://ledgerhq.atlassian.net/browse/LIVE-9901?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-mobile/src/families/elrond/accountActions.tsx [hunk 1]
```diff
@@ -39,11 +39,10 @@ const getMainActions = ({
    */
   const preloaded = getCurrentElrondPreloadData();
   const validators = randomizeProviders(preloaded.validators);
-
-  const screen =
-    account.elrondResources && account.elrondResources.delegations.length === 0
-      ? ScreenName.ElrondDelegationStarted
-      : ScreenName.ElrondDelegationValidator;
+  const firstTimeFlow = account.elrondResources && account.elrondResources.delegations.length === 0;
+  const screen = firstTimeFlow
```
Comment (beths-ledger) [inline apps/ledger-live-mobile/src/families/elrond/accountActions.tsx:43]: Generally i think it's clearer when booleans are `is` or `should` prefixed, e.g. `isFirstTimeFlow` would be super clean here (IMO)

File: apps/ledger-live-mobile/src/families/near/accountActions.tsx [hunk 2]
```diff
@@ -19,6 +19,9 @@ const getMainActions = ({
   parentRoute: RouteProp<ParamListBase, ScreenName>;
 }): ActionButtonEvent[] => {
   const stakingDisabled = !canStake(account);
+  const startWithValidator =
```
Comment (beths-ledger) [inline apps/ledger-live-mobile/src/families/near/accountActions.tsx:22]:
```suggestion
  const shouldStartWithValidator =
```
(opinion, take or leave)

File: apps/ledger-live-mobile/src/families/elrond/components/Flows/Delegate/types.ts [hunk 3]
```diff
@@ -20,6 +20,7 @@ export type ElrondDelegationFlowParamList = {
     account: ElrondAccount;
     transaction?: Transaction;
     source?: RouteProp<ParamListBase, ScreenName>;
+    trackingFirstStep?: boolean;
```
Comment (beths-ledger) [inline apps/ledger-live-mobile/src/families/elrond/components/Flows/Delegate/types.ts:23]:
```suggestion
    isFirstStep?: boolean;
```
or `trackAsFirstStep`

**Review by beths-ledger — COMMENTED** [review]
Looks good. I left a few minor ideas about variable names, otherwise the only issue to resolve seems to be the inconsistency in the currency value between `id` and `ticker`!

File: apps/ledger-live-mobile/src/families/elrond/accountActions.tsx (hunk 1 above)
Comment (mitchellv-ledger) [reply apps/ledger-live-mobile/src/families/elrond/accountActions.tsx:43]: agree with adding `is`

**Review by beths-ledger — DISMISSED** [review]
If the tracking doc requires a combination of capitalised names, ids and tickers for tracking currencies then so be it

**Review by haammar-ledger — DISMISSED** [review]

**Review by beths-ledger — APPROVED** [review]

**Pull request merged**